src/Tasks/Microsoft.NET.Build.Tasks/ClsidMap.cs

```diff
@@ -189,12 +187,46 @@ private static string GetTypeGuid(MetadataReader reader, TypeDefinition type)
                 if (reader.StringComparer.Equals(attributeType.Namespace, "System.Runtime.InteropServices") && reader.StringComparer.Equals(attributeType.Name, "GuidAttribute"))
                 {
                     CustomAttributeValue<KnownType> data = attribute.DecodeValue(new TypeResolver());
-                    return (string)data.FixedArguments[0].Value;
+                    return $"{{{(string)data.FixedArguments[0].Value}}}";
```

Comment:
I am curious, what's the reason for this change?

Reply:
These GUIDs should be wrapped in curly brackets. It's something I noticed they weren't while I was working on this.